docs/paper.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "antimeridian: A Python package for fixing geometries that cross the 180th meridian"
+tags:
+  - Python
+  - geospatial
+  - antimeridian
+authors:
+  - name: Peter Gadomski
+    orcid: 0000-0003-4877-7217
+    corresponding: true
+    affiliation: 1
+  - name: Preston Hartzell
+    orcid: 0000-0002-8293-3706
+    affiliation: 2
+affiliations:
+  - name: Development Seed, USA
+    index: 1
+  - name: Element 84, Inc., USA
+    index: 2
+date: 10 October 2024
+bibliography: paper.bib
+---
+
+## Summary
+
+Locations on and around planet Earth can be represented in a geodetic coordinate system by a latitude, a longitude, and a height.
+Longitude, the "horizontal" dimension, covers the domain from -180° to 180° or 0° and 360°.
+Where the two domain bounds meet is known as the _180th meridian_ or the _antimeridian_.
+
+![Earth map centered on the Pacific ocean, with the 180th meridian highlighted.](./img/antimeridian.jpg)
+
+The GeoJSON specification [@10.17487/RFC7946] describes how antimeridian-crossing shapes should be represented.
+For a variety of reasons, real-world geometries often do not comply with the specification, leading to confusing and unrepresentable geometries.
```

Comment:
_Maybe_ we can get away with "for a variety of reasons" here in the Summary, but this is vague verbiage and will likely get flagged.